src/state/opamSwitchState.ml

```diff
@@ -858,6 +858,14 @@ let universe st
     | Some set -> set
     | None -> OpamPackage.Set.empty
   in
+  let missing_depexts =
+    OpamPackage.Map.fold (fun nv status acc ->
+        if OpamSysPkg.Set.is_empty status.OpamSysPkg.s_available
```

Comment:
Should'nt be if available and not found are empty?

Reply:
Not necessarily, as not found one aren't in the request (removed from available)